cmd/machine-api-operator/start.go

```diff
@@ -139,7 +152,12 @@ func startControllers(ctx *ControllerContext) {
 		ctx.ClientBuilder.MachineClientOrDie(componentName),
 		ctx.ClientBuilder.DynamicClientOrDie(componentName),
 		recorder,
-	).Run(1, ctx.Stop)
+	)
+	if err != nil {
+		panic(fmt.Errorf("error creating operator: %v", err))
+	}
+
+	optr.Run(1, ctx.Stop)
```

Comment:
This used to be handled in a go routine right? I think this is blocking now and causing the operator to fail